tests/foreman/cli/test_host.py

```diff
@@ -2558,3 +2559,65 @@ def test_positive_dump_enc_yaml(default_sat):
     assert f'fqdn: {default_sat.hostname}' in enc_dump
     assert f'ip: {default_sat.ip_addr}' in enc_dump
     assert 'ssh-rsa' in enc_dump
+
+
+# -------------------------- HOST TRACE SUBCOMMAND SCENARIOS -------------------------
+def generate_custom_rhel_repo_string(major, minor):
+    domain = 'http://download.eng.bos.redhat.com'
+    if major == 8:
+        baseurl = (
+            f'{domain}/rhel-{major}/rel-eng/RHEL-{major}/'
+            f'latest-RHEL-{major}.{minor}/compose/AppStream/x86_64/os/'
+        )
+    elif major == 7:
+        baseurl = (
+            f'{domain}/rhel-{major}/rel-eng/RHEL-{major}/'
+            f'latest-RHEL-{major}.{minor}/compose/Server/x86_64/os/'
+        )
+    else:
+        raise ValueError('not supported major version')
+    return f"""
+    [custom-rhel]
+    name=custom-rhel
+    baseurl={baseurl}
+    enabled=1
+    gpgcheck=0
+    """
+
+
+@pytest.mark.tier3
+def test_positive_traces(katello_host_tools_client, default_sat, katello_host_tools_repos):
```

Comment:
this is blocked by introduction of helper methods mentioned in #9113
and just created issue https://github.com/SatelliteQE/robottelo/issues/9202

you will install the tracer anyway, only question is where, and agree with you it can be in some fixture,  it's better  code immersion

Reply:
I do not want to block this MR on this. Feel free to resolve this thread.